--- mp3packer-1.23/makefile.linking	2012-01-11 16:53:07.741139917 +0100
+++ mp3packer-1.23/makefile	2012-01-11 16:54:28.646072705 +0100
@@ -12,7 +12,7 @@ mp3packer: crc.cmx list2.cmx expandarray
 #	ocamlopt -o mp3packer unix.cmxa str.cmxa crc.cmx expandarray.cmx mp3types.cmx pack.cmx mp3read.cmx mp3info.cmx mp3framehuffman.cmx mp3queue.cmx mp3packer.ml
 
 mp3reader: crc.cmx list2.cmx c_part$(OBJ_EXT) mp3types.cmx pack.cmx mp3read.cmx mp3reader.ml
-	ocamlopt -o mp3reader$(EXE_EXT) unix.cmxa crc.cmx list2.cmx c_part$(OBJ_EXT) mp3types.cmx pack.cmx mp3read.cmx mp3reader.ml
+	ocamlopt -o mp3reader$(EXE_EXT) unix.cmxa str.cmxa crc.cmx list2.cmx c_part$(OBJ_EXT) mp3types.cmx pack.cmx mp3read.cmx mp3reader.ml
 
 
 mp3framehuffman.cmx: mp3types.cmx pack.cmx mp3framehuffman.ml
